tools/donate-cpu.py

```diff
@@ -542,10 +562,11 @@ def uploadInfo(package, info_output, server_address):
         print(output)
         print('=========================================================')
         break
```

Comment:
We could remove the `break` here. `max_packages` is set to 1 now when the `--package` option is specified, so the script stops anyway. And since there is now the option `--no-upload` it is possible to test a package without uploading it by using that option. When `--no-upload` is not specified the results are uploaded. IMHO that would be useful to update some package results for example after fixing a crash. What do you think?